### ISSUE TITLE
Parse password

### ISSUE THREAD

**Issue by logan-r** [HTML5-Games/html5-games.github.io#3]
If anybody needs the password to our [Parse](http://parse.com) account, just post a way I can send it to you.

**Comment by Nikolas-Lennon** [comment]
what is the password?

**Comment by Nikolas-Lennon** [comment]
Yeah I have skype, nikolas.lennon is my username.

**Comment by logan-r** [comment]
@williamvanr I sent you the password.
@Tritexander I don't have skype. Do you have an email address I can send it to?

**Comment by Nikolas-Lennon** [comment]
Tritexander07@gmail.com

**Comment by logan-r** [comment]
Sent.

**Comment by MachoLuke** [comment]
send to macholucas12@gmail.com

**Comment by logan-r** [comment]
Also sent.

**Comment by Nikolas-Lennon** [comment]
Okay, should we add a feature that allows people who are not signed up with
us to just view posts and not make posts? That way it gives us a chance to
gain more members to the site? And the ranks, should Admin just be a thing
given by other admins, and the highest be a Moderator of posts?

**Comment by ferndot** [comment]
@Logi0 can I have it too?

**Comment by TheSentinel387** [comment]
@tritexander I was thinking maybe moderator could look after the
forums/posts more and there would only be 5-10 admins who would look after
the whole website/server but would still have the mod permissions. Perhaps
anonymous people (not logged in/signed up) can only see certain posts,
which would have to be flagged by a mod/admin, then redirected to login or
register on any other posts?

On 9 June 2013 09:15, Joshua Smith notifications@github.com wrote:

> @Logi0 https://github.com/Logi0 can I have it too?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/HTML5-Games/html5-games.github.io/issues/3#issuecomment-19158939
> .

## 

++aLL Is DuSt++

**Comment by TheSentinel387** [comment]
also 1021817@student.ccgs.wa.edu.au

On 9 June 2013 11:18, Joseph Kuek 1021817@student.ccgs.wa.edu.au wrote:

> @tritexander I was thinking maybe moderator could look after the
> forums/posts more and there would only be 5-10 admins who would look after
> the whole website/server but would still have the mod permissions. Perhaps
> anonymous people (not logged in/signed up) can only see certain posts,
> which would have to be flagged by a mod/admin, then redirected to login or
> register on any other posts?
> 
> On 9 June 2013 09:15, Joshua Smith notifications@github.com wrote:
> 
> > @Logi0 https://github.com/Logi0 can I have it too?
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/HTML5-Games/html5-games.github.io/issues/3#issuecomment-19158939
> > .
> 
> ## 
> 
> ++aLL Is DuSt++

## 

++aLL Is DuSt++

**Comment by Nikolas-Lennon** [comment]
Yes, precisely. That is what I was thinking. But the way we are using this
comment system, it should be based of a forum type instead a blog type. I
think we should transform the blog into a forum instead because I believe
that it would be easier to locate posts, it would be easier to navigate
through, and easier to moderate. But it isn't up to me. With all the
categories we have that are available to choose from, we could easily add
sub categories and further the development of the website.

**Comment by TheSentinel387** [comment]
@tritexander yeah I think the blog should remain purely for updates/patches
and limited to mods/admins only, but we should have a 'separate' forum.

On 9 June 2013 11:23, Tritexander notifications@github.com wrote:

> Yes, precisely. That is what I was thinking. But the way we are using this
> comment system, it should be based of a forum type instead a blog type. I
> think we should transform the blog into a forum instead because I believe
> that it would be easier to locate posts, it would be easier to navigate
> through, and easier to moderate. But it isn't up to me. With all the
> categories we have that are available to choose from, we could easily add
> sub categories and further the development of the website.
> 
> On Sat, Jun 8, 2013 at 10:19 PM, TheSentinel387 notifications@github.comwrote:
> 
> > @tritexander I was thinking maybe moderator could look after the
> > forums/posts more and there would only be 5-10 admins who would look
> > after
> > the whole website/server but would still have the mod permissions.
> > Perhaps
> > anonymous people (not logged in/signed up) can only see certain posts,
> > which would have to be flagged by a mod/admin, then redirected to login
> > or
> > register on any other posts?
> > 
> > On 9 June 2013 09:15, Joshua Smith notifications@github.com wrote:
> > 
> > > @Logi0 https://github.com/Logi0 can I have it too?
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub<
> > 
> > https://github.com/HTML5-Games/html5-games.github.io/issues/3#issuecomment-19158939>
> > 
> > > .
> > 
> > ## 
> > 
> > ++aLL Is DuSt++
> > 
> > —
> > Reply to this email directly or view it on GitHub<
> > https://github.com/HTML5-Games/html5-games.github.io/issues/3#issuecomment-19160068>
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/HTML5-Games/html5-games.github.io/issues/3#issuecomment-19160104
> .

## 

++aLL Is DuSt++

**Comment by Nikolas-Lennon** [comment]
We would need a MySQL server and php to create a forum though for it to be
good. Which we don't have :(

**Comment by logan-r** [comment]
@Joshua-S, @TheSentinel387 Sent.
@Tritexander We don't need MySQL, we have Parse.

**Comment by Maartenf** [comment]
And what about SEO?

**Comment by Nikolas-Lennon** [comment]
I keep forgetting about that... haha I am so sorry. I am not used to this
Parse stuff yet.

**Comment by Nikolas-Lennon** [comment]
I'm not sure I understand how to get started with making a forum with
Parse. It isn't making sense to me. All I can find of Parse is that it
hosts Apps on Windows 8 and iphones and androids.

**Comment by mektrode** [comment]
Yes, i too would like the password sent to "mowz971@gmail.com". Also, Logan is there a place i can learn (tutorials) to start using Parse for development. Especially the user login system. I'm intrigued!

**Comment by logan-r** [comment]
@M-Hc0d3 Sorry for taking so long, but it's sent now. Codecademy has some tutorials in the API Track. Also checkout the [Parse documentation](https://www.parse.com/docs/js_guide).

**Comment by codypete1114** [comment]
I could use the Parse password, email is: codypete1114@gmail.com
